moped-editor/src/views/projects/projectView/ProjectSummary/TagsSection.js

```diff
@@ -82,7 +82,7 @@ const TagsSection = ({ projectId }) => {
    * Filter out already selected tags
    */
   const availableTags = () => {
-    const tagsIds = data.moped_proj_tags.map((t) => t.id);
+    const tagsIds = data.moped_proj_tags.map((t) => t.moped_tag.id);
```

Comment:
ensures duplicate tags cannot be added to a project. thanks for remembering this! i completely forgot.